src/style.css

```
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  --blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__image {
+  width: 100%;
+  height: 100%;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  white-space: nowrap;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  transition: color 0.3s;
+}
+
+.nav__link:hover {
+  color: var(--blue);
+}
+
+.is-active {
+  color: var(--blue);
+}
+
+.is-active::after {
+  display: block;
+  position: absolute;
+  height: 4px;
+  background-color: var(--blue);
+  border-radius: 2px;
+  content: "";
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

Just the order of the rules 

```suggestion
.is-active::after {
  content: ""; // nothing will work without it
  
  // All position rules
  position: absolute;
  bottom: 0;
  left: 0;
  right: 0;
  
  // Box model rules
  display: block;
  height: 4px;
  
  // text styles if exist
  
  // other styles
  background-color: var(--blue);
  border-radius: 2px;
}
```

---

Thank you for the notes. All finished...